functionary/builder/management/commands/generate_package_schema.py

```diff
@@ -0,0 +1,42 @@
+import json
+import pathlib
+
+from django.core.management.base import BaseCommand
+from drf_jsonschema_serializer import to_jsonschema
+
+from builder.api.v1.serializers.package_definition import (
+    PackageDefinitionWithVersionSerializer,
+)
+
+
+class Command(BaseCommand):
+    help = "Generates the schema.json used to validate the package.yaml"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-o",
+            "--output",
+            action="store",
+            dest="schema_file",
+            type=pathlib.Path,
+            nargs=1,
+            help="Output file to write the schema to",
+        )
+
+    def handle(self, *args, **options):
+        json_schema = to_jsonschema(PackageDefinitionWithVersionSerializer())
+        json_schema["$schema"] = "https://json-schema.org/draft/2020-12/schema"
+        json_schema["$id"] = "https://functionary/package.yaml"
```

Comment:
I don't think `$id` is required, so probably just leave it out rather than putting this fake url in there.  The [jsonschema spec](https://json-schema.org/draft/2020-12/json-schema-core.html#name-the-id-keyword) says:

> If present, the value for this keyword MUST be a string, and MUST represent a valid [URI-reference](https://json-schema.org/draft/2020-12/json-schema-core.html#RFC3986) [[RFC3986](https://json-schema.org/draft/2020-12/json-schema-core.html#RFC3986)].

Reply:
Removed